hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/datanode/datanode.html

```diff
@@ -84,7 +84,8 @@
       <th>Namenode HA State</th>
       <th>Block Pool ID</th>
       <th>Actor State</th>
-      <th>Last Heartbeat</th>
+      <th>Last Heartbeat Sent</th>
+      <th>Last Heartbeat Received</th>
```

Comment:
Last Heartbeat Received gives me a sense like Namenode sent heartbeat to Datanode, That ain't good from Correctness point of view.
Can you change to indicate it is the heartbeat response rather than the heartbeat itself